src/beignet/_ranked_fft.py

```diff
@@ -0,0 +1,77 @@
+import edlib
+import heapq
+import numpy as np
+from typing import Optional
+
+
+def edit_dist(x: str, y: str):
+    """
+    Computes the edit distance between two strings.
+    """
+    return edlib.align(x, y)["editDistance"]
```

Comment:
I don’t want to carry edlib unless (or until) it’s absolutely necessary. 

I believe edlib uses Myers’. I could implement it if you don’t have the time. We could also use Levenshtein.

Reply:
yep, gotta keep dependencies under control. This is maybe another reason to change to an arbitrary distance function and let the user decide what they want to use. I used to use Levenshtein and then switched to edlib for reasons I don't remember

Reply:
for now I've punted by just adding a `distance_fn` arg